gravitee-apim-console-webui/src/management/api/creation/v4/steps/step-6/api-creation-v4-step-6.component.ts

```diff
@@ -29,11 +34,48 @@ import { ApiCreationStepService } from '../../services/api-creation-step.service
 })
 export class ApiCreationV4Step6Component {
   private unsubscribe$: Subject<void> = new Subject<void>();
-  constructor(private readonly stepService: ApiCreationStepService, private readonly matDialog: MatDialog) {}
+  constructor(
+    private readonly stepService: ApiCreationStepService,
+    private readonly matDialog: MatDialog,
+    private readonly snackBarService: SnackBarService,
+    private readonly apiV4Service: ApiV4Service,
+  ) {}
 
   createApi(): void {
-    alert(this.stepService.payload);
-    // TODO: send info to correct endpoint to create, not publish, the new API
+    const apiCreationPayload = this.stepService.payload;
+    // eslint-disable-next-line
+    console.info('API Creation Payload', apiCreationPayload);
+
+    this.apiV4Service
+      .create(
```

Comment:
Note we need to add test to this call. but we can do it when the step 6 was more "concret"

Reply:
Ok! We can test this after rebasing.